templates/max/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "{{project-name}}"
+name = "project-name"
 version = "0.1.0"
-authors = [ "{{ authors }}" ]
+authors = [ "authors" ]
 edition = "2018"
 
 [lib]
-name = "{{crate_name}}"
+name = "crate_name"
```

Comment:
you removed the substitutions, did that change somehow??

Reply:
`cargo generate --git https://github.com/eilseq/median.git --branch develop --name my-project`

produces `Cargo.toml`

```
[package]
name = "project-name"
version = "0.1.0"
authors = [ "authors" ]
edition = "2018"

[lib]
name = "crate_name"
crate-type = ["cdylib"]

[dependencies]
median = { git = "https://github.com/Cycling74/median" }

[profile.release]
lto = true
panic = "abort"
```

so this change breaks the template

Reply:
Hi @x37v, the minimal changes I have made was so I could succesfully complie using my pipeline on this project.
Without these modifications was failing. I cannot address additional time to this right now but happy to share results from destination project:


https://github.com/eilseq/Dice